editor/plugins/script_editor_plugin.cpp

```diff
@@ -1236,14 +1237,15 @@ void ScriptEditor::_menu_option(int p_option) {
 			_update_script_names();
 		} break;
 		case TOGGLE_SCRIPTS_PANEL: {
+			toggle_scripts_panel();
 			if (current) {
-				ScriptTextEditor *editor = Object::cast_to<ScriptTextEditor>(current);
-				toggle_scripts_panel();
-				if (editor) {
-					editor->update_toggle_scripts_button();
-				}
+				current->update_toggle_scripts_button();
 			} else {
-				toggle_scripts_panel();
+				Node *tab = tab_container->get_current_tab_control();
```

Comment:
Shouldn't really matter but it returns a `Control *` (didn't notice that previously):
```suggestion
				Control *tab = tab_container->get_current_tab_control();
```